test/run_ddt_tests.lua

```diff
@@ -152,6 +152,8 @@ local function compile_stage(test, args)
     local compile_opts          = test.schema
     compile_opts.service_fields = service_fields
     compile_opts.downgrade      = compile_downgrade
+    -- would be deleted after #85
+    compile_opts.alpha_nullable_record_xflatten = true
```

Comment:
That should not be part of that commit.

Reply:
What do you mean?

Reply:
You added this certain hunk outside of the commit where all other hunks related to the flag were added.

Reply:
fixed